lesson9/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Recap & Practicing OOP
+nav_order: 9
+has_children: false
+nav_exclude: false
+---
+
+# Lesson 8: Recap & Practicing OOP
+
+## Goals
+
+- Check-in
+- Recap OOP
+
+## Check-in
+How is everybody doing?
+
+## Recap homework and previous class.
```

Comment:
Didn't we have this last time?

Reply:
I guess the class happened without a lesson in the crouse as it is a recap.